### DISCUSSION TITLE
Lession 11: ERC20Mocks.sol

### DISCUSSION THREAD

**Discussion by AztecWarlord** [Cyfrin/foundry-full-course-cu#432] [General]
I had to copy and paste the ERC20Mocks.sol from the actual repo in Open Zeppelin because the installed version downloaded a different version and was breaking my code. Am I the only one who had this problem?

**Comment by shikhar229169** [comment]
Yea everyone faced this problem. Because in the latest version of open zeppelin, the code is changed. So, I recommend to install the required version. You can check it out by visiting the required repo in Cyfrin.
